tests/regression/evaluate_rag_retrieval.py

```
@@ -82,15 +174,51 @@ def hallucination_flags(answer: str, hits, expected_url_fragments: list[str]) ->
 
 
 def evaluate_search_case(case: dict, top_k: int, answer_top_k: int) -> dict:
-    from LLM.sub_model.query_index import build_answer, hybrid_search
-
     query = case.get("query", "")
+    try:
+        from LLM.sub_model.query_index import build_answer, hybrid_search
+        import_error = ""
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)